.github/workflows/bench.yml

```diff
@@ -11,7 +11,6 @@ concurrency:
 
 env:
   NODE_NO_WARNINGS: 1
-  CI: true
```

Comment:
[GitHub sets `CI=true` since 2020.](https://github.blog/changelog/2020-04-15-github-actions-sets-the-ci-environment-variable-to-true/)